ThumbnailItem/ThumbnailItem.js

```diff
@@ -13,25 +13,76 @@ import kind from '@enact/core/kind';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import componentCss from './ThumbnailItem.module.less';
 import Item from '../Item';
 
+import componentCss from './ThumbnailItem.module.less';
 
 /**
  * A stateless, unfocusable item that can display a thumbnail.
  *
  * @class ThumbnailItemBase
  * @memberof agate/ThumbnailItem
  * @extends agate/Item.Item
- * @agate
+ * @ui
  * @public
  */
 const ThumbnailItemBase = kind({
 	name: 'ThumbnailItem',
 
 	propTypes: /** @lends agate/ThumbnailItem.ThumbnailItemBase.prototype */ {
+		/**
+		 * The main content displayed with the thumbnail.
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		children: PropTypes.string,
+
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * * `thumbnailItem` - The root component class
+		 * * `content` - The content component class
+		 * * `subContent` - The subContent component class
+		 * * `roundThumbnail` - Applied when `roundThumbnail="true"`
+		 * * `thumbnail` - The thumbnail component class
+		 *
+		 * @type {Object}
+		 * @public
+		 */
 		css: PropTypes.object,
-		src: PropTypes.string
+
+		/**
+		 * Applies `selected` className.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		selected: PropTypes.bool,
+
+		/**
+		 * String value used to determine which thumbnail will appear on a specific screenSize.
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		src: PropTypes.string,
+
+		/**
+		 * The thumbnail type.
+		 *
+		 * @type {('round'|'square')}
+		 * @default 'square'
+		 * @public
+		 */
+		type: PropTypes.oneOf(['round', 'square'])
```

Comment:
Naming ideas:
* `normal` and `styled`
* `normal` and `stylized`
* `normal` and `accent`

Reply:
Renamed `square` -> `normal` and `round` -> `styled`